sense/feature_extractors/mobilenet.py

```diff
@@ -192,7 +192,7 @@ def forward(self, video):
         return self.cnn(video)
 
     def preprocess(self, clip):
-        clip = clip[:, :, :, ::-1].copy()
```

Comment:
I think the `copy` is no longer necessary, because it was probably only used here to resolve the `::-1`